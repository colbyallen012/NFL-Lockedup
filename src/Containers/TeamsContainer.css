.team-container {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
}

.team-container { grid-area: 3 / 2 / 3 / 6; }
