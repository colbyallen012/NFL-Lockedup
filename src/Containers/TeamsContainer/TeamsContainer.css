.team-container {
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  justify-content: center;
  margin: 5px auto;
  width: 86%;
}

