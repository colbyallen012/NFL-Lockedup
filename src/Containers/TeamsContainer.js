import React from 'react'; 
import Team from '../Components/Team'
import './TeamsContainer.css'


const TeamsContainer = ({teams}) => {
  const displayTeams = teams.map(team => {
    return (
      <Team 
        city={team.Team_city}
        name={team.Team_name}
        conference={team.Team_Conference}
        abrv={team.Team}
        key={team.Team}
      />
    )
  })
  return (
    <section className='team-container'>
      <section className='team-container'>
        {displayTeams}
      </section>
    </section>
  )
}

export default TeamsContainer;