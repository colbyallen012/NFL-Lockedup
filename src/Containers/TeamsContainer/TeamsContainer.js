import React from 'react'; 
import Team from '../../Components/Team/Team'
import PropTypes from 'prop-types';
import './TeamsContainer.css'


const TeamsContainer = ({arrests}) => {
  const displayTeamArrests = arrests.map(arrest => {
    return (
      <Team 
        player={arrest.Name}
        position={arrest.Position_name}
        arrest={arrest.Category}
        description={arrest.Description}
        outcome={arrest.Outcome}
        season={arrest.ArrestSeasonState}
        key={arrest.Arrest_stats_id}
      />
    )
  })
  return (
    <section className='team-container'>
      <section className='team-container'>
        {displayTeamArrests}
      </section>
    </section>
  )
}

TeamsContainer.propTypes = {
  arrests: PropTypes.array.isRequired
}

export default TeamsContainer;