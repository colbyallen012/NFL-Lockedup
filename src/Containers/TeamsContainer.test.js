import { shallow } from 'enzyme';
import React from 'react';
import TeamsContainer from './TeamsContainer';

describe('TeamsContainer', () => {
  let wrapper;
  const mockTeam = [{
    arrest_stats_id: "903",
    Date: "2018-10-22",
    Team: "DEN",
    Team_name: "Broncos",
    Team_preffered_name: "Denver Broncos",
    Team_city: "Denver",
    Team_logo_id: "10",
    Team_Conference: "AFC",
    Team_Division: "West",
    Team_Conference_Division: "AFC West",
    Team_hex_color: "FC4C02",
    Team_hex_alt_color: "0C2340",
    reddit_group_id: "3",
    Name: "Chad Kelly",
    Position: "QB",
    Position_name: "Quarterback",
    Position_type: "O",
    Encounter: "Arrested",
    Category: "Trespassing",
    Crime_category: "Other",
    Crime_category_color: "BCBD22",
    Description: "The Englewood Colorado Police arrested and charged Kelly with first-degree criminal trespassing after a report of a man allegedly inside a home.",
    Outcome: "Resolution Undetermined.",
    Season: "2018",
    ArrestSeasonState: "InSeason",
    overall_category_id: "15",
    general_category_id: "15",
    legal_level_id: "1",
    resolution_category_id: "1",
    Year: "2018",
    Month: "10",
    Day: "22",
    Day_of_Week: "Monday",
    Day_of_Week_int: "2",
    YearToDate: "0",
    DaysSince: "244",
    DaysToLastArrest: "36",
    DaysToLastCrimeArrest: null,
    DaysToLastTeamArrest: "229"
  }]

  beforeEach(() => {
    wrapper = shallow(<TeamsContainer arrests={mockTeam} key={1} />)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  }); 
})