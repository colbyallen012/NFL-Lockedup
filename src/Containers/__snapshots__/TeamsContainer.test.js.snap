// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamsContainer should match snapshot 1`] = `
<section
  className="team-container"
>
  <section
    className="team-container"
  >
    <Team
      arrest="Trespassing"
      description="The Englewood Colorado Police arrested and charged Kelly with first-degree criminal trespassing after a report of a man allegedly inside a home."
      outcome="Resolution Undetermined."
      player="Chad Kelly"
      position="Quarterback"
      season="InSeason"
    />
  </section>
</section>
`;
